app/models/mission.rb

```diff
@@ -1,4 +1,20 @@
 class Mission < ApplicationRecord
-  belongs_to :mentor
-  belongs_to :padawan
+  belongs_to :mentor, class_name: "User"
+  belongs_to :padawan, class_name: "User", optional: true
+
+  enum status: [ :not_started, :started, :finished ]
```

Comment:
We think it would be better to update " :started" by "on-going"

Reply:
Done in next commit